ts/state/selectors/user.ts

```diff
@@ -13,3 +14,8 @@ export const getOurNumber = createSelector(
 );
 
 export const getIntl = createSelector(getUser, (): LocalizerType => window.i18n);
+
+export const getHTMLDirection = createSelector(
+  getUser,
+  (): HTMLDirection => (isRtlBody() ? 'rtl' : 'ltr')
+);
```

Comment:
I'm sorry... I don't like it like this neither, as it doesn't return something part of the redux state. It's just a function.

I think we could do something like that instead
```ts

export const useHTMLDirection =   (): HTMLDirection => (isRtlBody() ? 'rtl' : 'ltr')
```

and then make

```ts

export const SessionMainPanel = () => {
  const focusedSettingsSection = useSelector(getFocusedSettingsSection);
  const isSettingsView = focusedSettingsSection !== undefined;
  const htmlDirection = useHTMLDirection()

  // even if it looks like this does nothing, this does update the redux store.
  useAppIsFocused();

  if (isSettingsView) {
    return <FilteredSettingsView category={focusedSettingsSection} />;
  }
  return (
    <div className="session-conversation">
      <SmartSessionConversation htmlDirection={htmlDirection} />
    </div>

  );
};

```

Reply:
Fair enough but to point out we should probably update `getIntl` based on that logic. It's pretty minor and only used in the ConverationComparator

```
export const getIntl = createSelector(getUser, (): LocalizerType => window.i18n);
```

Will fix now

Reply:
Fixed I think. Hope I did this correctly. I used https://react-redux.js.org/api/connect as a reference.